packages/squiggle-lang/src/rescript/Distributions/SymbolicDist/SymbolicDist.res

```diff
@@ -80,8 +80,8 @@ module Triangular = {
   let make = (low, medium, high): result<symbolicDist, string> =>
     low < medium && medium < high
       ? Ok(#Triangular({low: low, medium: medium, high: high}))
-      : Error("Triangular values must be increasing order")
-  let pdf = (x, t: t) => Jstat.Triangular.pdf(x, t.low, t.high, t.medium)
+      : Error("Triangular values must be increasing order.")
+  let pdf = (x, t: t) => Jstat.Triangular.pdf(x, t.low, t.high, t.medium) // not obvious in jstat docs that high comes before medium? 
```

Comment:
(responding to comment)
It seems to work like we'd expect, but maybe something super weird is happening.